File: app/src/main/java/com/example/mettest/models/Status.java

package com.example.mettest.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Status {

    @SerializedName("error_code")
    @Expose
    public Integer errorCode;
    @SerializedName("message")
    @Expose
    public String message;

}

File: app/src/main/java/com/example/mettest/models/Status.kt
package com.example.mettest.models

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import com.example.mettest.models.Datum

class Status {
    @SerializedName("error_code")
    @Expose
    var errorCode: Int? = null

    @SerializedName("message")
    @Expose
    var message: String? = null
}